dev/merge_arrow_pr.py

```diff
@@ -110,10 +111,32 @@ def strip_ci_directives(commit_message):
     return _REGEX_CI_DIRECTIVE.sub('', commit_message)
 
 
+def git_default_branch_name():
+    default_branch_name = os.getenv("MERGE_SCRIPT_DEFAULT_BRANCH_NAME")
+
+    if default_branch_name is None:
+        try:
+            default_reference = run_cmd(
+                "git rev-parse --abbrev-ref origin/HEAD")
+            default_branch_name = default_reference.lstrip("origin/")
```

Comment:
Do we need `rstrip()` here to remove new line?

Reply:
Thank you for your suggestion @kou. There is a newline character that I've added an `rstrip()` call to remove.

Reply:
This is a common misunderstanding of strip()/lstrip()/rstrip().
See https://docs.python.org/3/library/stdtypes.html#str.lstrip
You need something like:
```suggestion
            if default_branch_name.startswith("origin/"):
                default_branch_name = default_branch_name[len("origin/"):]
```